src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/HmppsDomainEventEmitter.kt

```diff
@@ -130,11 +130,12 @@ class HmppsDomainEventEmitter(
 
   fun emitPrisonerAlertsUpdatedEvent(
     offenderNo: String,
+    bookingId: String?,
```

Comment:
How come this is nullable?

Reply:
Because I am LAZY, it is nullable in the offender record (new offenders have no bookings) so just allowed it to be nullable through out knowing full well in this context it will always be present, but I don't care if it isn't